Services/UICore/classes/class.ilTemplate.php

```diff
@@ -1521,6 +1521,18 @@ function getTemplatePath($a_tplname, $a_in_module = false, $a_plugin = false)
 				$fname = "./".$module_path."templates/default/".basename($a_tplname);
 			}
 		}
+		else if(strpos($a_tplname,"src/UI")===0)
```

Comment:
Thank you for your feedback, and interest in this PR!

You mean directly inside the code or here in the disucssion?

Those lines are needed to enable templating for skins of templates from the centralized UI components. The logic is to check, wether the folder currentSkin/UI/pathToTemplate/tpl.name_of_template.html and if so, use this.